Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM matomo:latest
-
-# Install system dependencies
-RUN apt-get update && apt-get upgrade -y 
-
-
-EXPOSE 80 443
-
-COPY config.ini.php /var/www/html/config/config.ini.php
-
-
+FROM matomo:3.13.6-apache
 
+# Add the EnvironmentVariables plugin
+COPY ./files/plugin-EnvironmentVariables-3.0.0/ /var/www/html/plugins/EnvironmentVariables
 
+# Preconfigure settings
+COPY ./files/config.ini.php /var/www/html/config/config.ini.php
 
+# Create mount point for EFS partition
+RUN mkdir -p /mnt/efs
 
+EXPOSE 80
```

Comment:
Are we terminating SSL further up the stack than the container?

Reply:
Yes, SSL gets terminated at the ALB. Over in the mitlib-tf-workloads-matomo infrastructure repository, you can see the listener rules on the ALB that redirect anything coming in on port 80 to port 443.
* https://github.com/MITLibraries/mitlib-tf-workloads-matomo/blob/906911ce80fab42e8704ca0dabb5bc00c846f714/network.tf#L35
* https://github.com/MITLibraries/mitlib-tf-workloads-matomo/blob/906911ce80fab42e8704ca0dabb5bc00c846f714/network.tf#L57
*